VMWare guest utils depend on current Linux kernel, but since the installed kernel might not be
up-to-date, it is best that we prevent the guest utils from being upgraded during system upgrades.

--- /etc/pacman.conf
+++ /etc/pacman.conf
@@ -23,7 +23,7 @@
 Architecture = auto

 # Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
-IgnorePkg    = linux linux-firmware linux-headers
+IgnorePkg    = linux linux-firmware linux-headers open-vm-tools
 #IgnoreGroup =

 #NoUpgrade   =
